CHANGELOG.md

```diff
@@ -1,24 +1,53 @@
-## 2.4 - 2024-06-25
-- [GRD-797](https://jira.oicr.on.ca/browse/GRD-797) - add vidarr labels to outputs (changes to medata only)
-## 2.3 2023-06-13
-- Move assembly-specific settings into the wdl
-## 2.2 2023-02-23
-- Update STAR to 2.7.10b
-- Change HardClip to Softclip for Chimeric Reads
-## 2.1.1 2021-05-31
-- Migrate to Vidarr
-## 2.1 - 2021-01-14
-- update STAR to 2.7.6a
-- add chimOutType option to write chimeric reads WithinBAM
-## 2.0.2 - 2020-05-04
-- Make --chimOutJunctionFormat parameter optional for versions of star older than 2.6.1a
-## 2.0.1 - 2020-01-13
-- Hotfix to expose --chimOutJunctionFormat parameter needed for starFusion
-## 2.0   - 2019-09-29
-- Converting to cromwell wdl with changes to provision junction file and take multiple fastq pairs
-## 1.1.1 - 2017-07-12
-- Fixed a bug with adapter sequences paramtrizing
-## 1.1   - 2016-10-03
-- Added new parameter produce_transcriptome_bam (true by default)
-## 1.0 - 2016-09-08
-- Initial import of STAR code
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [2.4] - 2024-06-25
```

Comment:
Three latest tags need to be fixed, these are three-digit tags, i.e. not 2.3 but 2.3.0